util/strutil/strconv.go

```diff
@@ -42,3 +44,11 @@ func GraphLinkForExpression(expr string) string {
 func SanitizeLabelName(name string) string {
 	return invalidLabelCharRE.ReplaceAllString(name, "_")
 }
+
+// IsValidLabelName evaluates whether a given label name is valid or not.
+func IsValidLabelName(name string) bool {
```

Comment:
Can we reuse this function in common/model to do the same? http://godoc.org/github.com/prometheus/common/model#LabelName.IsValid

Also for metric name validation this one: http://godoc.org/github.com/prometheus/common/model#IsValidMetricName